### ISSUE TITLE
List skills results

### ISSUE THREAD

**Issue by cvuorinen** [probaton/habotica#37]
Similar to how armoire results are shown, it would be nice to see the results of spamming skills. Especially ones that are not just buffs and gain XP, gold, health etc.

**Comment by probaton** [comment]
Wish I had thought of that! API docs claim the call returns result data, so with a bit of luck I'll be able to re-appropriate the result parser from the armoire code.

**Comment by probaton** [comment]
Apparently I spoke too soon. Rather than returning the actual results of the skill usage, the use skill API simply returns the entire user object. 

So to distill the effects of the skill, I would have to somehow subtract the old user data from the new. That seems a bit convoluted, considering all the possible fields that might be affected by a skill. 

Also, in my experience Habitica user data can be a bit flaky and hard to parse, so I would have to worry about whether the changes I'm seeing are due to the skill or some other factor. 

All in all I feel that the benefits of the feature don't warrant the extra complexity and brittleness, sorry!
